pkg/storage/system_intake.go

```diff
@@ -210,6 +210,22 @@ func (s *Store) FetchSystemIntakesByEuaID(ctx context.Context, euaID string) (mo
 	return intakes, nil
 }
 
+func generateLifecyclePrefix(t time.Time, loc *time.Location) string {
```

Comment:
Can you add a comment somewhere spelling out what the intended construction of the LCID is supposed to be? Having that as a reference point may help with future debugging.

Reply:
Will do!